src/Components/Event.php

```diff
@@ -244,6 +247,13 @@ public function status(EventStatus $status): Event
         return $this;
     }
 
+    public function uniformResourceLocator(string $url): Event
```

Comment:
Let's rename this to just `url`, although `uniformResourceLocator` is technically correct it doesn't feel natural to use.

Reply:
Okay, I will change this.
I thought of the long form, because of `uniqueIdentifier` ✌🏼

Reply:
Yeah, good catch though I think `uid` wasn't that clear and `uniqueIdentifier` would be, maybe we should add an alias for that in the future ...